contracts/contracts/ProjectRegistry.sol

```diff
@@ -59,7 +61,12 @@ contract ProjectRegistry {
         _;
     }
 
-    constructor() {}
+    /**
+     * @notice Initializes the contract after an upgrade
+     * @dev In future deploys of the implementation, an higher version should be passed to reinitializer
+     */
+    function initialize() public reinitializer(1) {
```

Comment:
I imagine we will be able to override `1` in subsequent initializations?

Reply:
yes exactly